src/parser/jobs/ast/index.tsx

```diff
@@ -29,12 +29,22 @@ export const ASTROLOGIAN = new Meta({
 	</>,
 	supportedPatches: {
 		from: '6.0',
-		to: '6.08',
+		to: '6.1',
 	},
 	contributors: [
-		{user: CONTRIBUTORS.OTOCEPHALY, role: ROLES.DEVELOPER},
+		{ user: CONTRIBUTORS.SUSHIROU, role: ROLES.MAINTAINER }, {user: CONTRIBUTORS.OTOCEPHALY, role: ROLES.DEVELOPER},
```

Comment:
GH will prob mangle the whitespace but you get the idea:
```suggestion
		{ user: CONTRIBUTORS.SUSHIROU, role: ROLES.MAINTAINER },
		{ user: CONTRIBUTORS.OTOCEPHALY, role: ROLES.DEVELOPER },
```